src/sentry/api/bases/organization.py

```diff
@@ -146,29 +145,27 @@ def get_projects(
 
         user = getattr(request, 'user', None)
 
-        if (
-            user and is_active_superuser(request)
-            or include_allow_joinleave and organization.flags.allow_joinleave
-            or force_global_perms
-        ):
-            qs = Project.objects.filter(
-                organization=organization,
-                status=ProjectStatus.VISIBLE,
-            )
-        else:
-            qs = Project.objects.filter(
-                organization=organization,
-                teams__in=OrganizationMemberTeam.objects.filter(
-                    organizationmember__user=user,
-                    organizationmember__organization=organization,
-                ).values_list('team'),
-                status=ProjectStatus.VISIBLE,
-            )
+        qs = Project.objects.filter(
+            organization=organization,
+            status=ProjectStatus.VISIBLE,
+        )
 
         if project_ids:
             qs = qs.filter(id__in=project_ids)
 
-        projects = list(qs)
+        if force_global_perms:
+            projects = list(qs)
+        else:
+            if (
+                user and is_active_superuser(request) or
+                requested_projects or
+                include_allow_joinleave
```

Comment:
Wondering if we should rename this to `include_all_accessible` or something, since we have a better idea of what it's used for now

Reply:
oops kind of forgot about this pr and i think perms are causing some issues for the support team. going to merge as is and follow up with the rename